ItemDescription3/tableEXT/salesLineTabExt.50401.TabExt.al

```diff
@@ -0,0 +1,12 @@
+tableextension 50401 appsalesLineTabExt extends "Sales Line"
```

Comment:
Please create AppSourceCop.json in the main folder with following structure:
{
    "mandatoryAffixes": ["YourMandatoryAffix"],
    "supportedCountries": ["US"]
}

Replace "YourMandatoryAffix" with some unique affix, "app" is not good enough. Let's use "BCP" as standard affix for all our tasks.

In this case naming will be "BCP Sales Line" or "BCP Description 3"

Good affix is mandatory to avoid any possible conflicts between names or future updates

Please update all your names with new affix

@Ams272